docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.11.8
+
+#clone archipelago
+RUN mkdir -p /Archipelago
+RUN git clone --depth 1 --branch 0.4.4 https://github.com/ArchipelagoMW/Archipelago.git /Archipelago
+#clone and unzip Enemizer
+RUN wget https://github.com/Ijwu/Enemizer/releases/latest/download/ubuntu.16.04-x64.zip;\
+    mkdir -p /Archipelago/EnemizerCLI;\
+    unzip -u ubuntu.16.04-x64.zip -d /Archipelago/EnemizerCLI;\
+    chmod -R 777 /Archipelago/EnemizerCLI;\
+    rm ubuntu.16.04-x64.zip
+#clone and untar SNI
+RUN wget https://github.com/alttpo/sni/releases/latest/download/sni-v0.0.95-linux-amd64.tar.xz;\
+    mkdir -p /Archipelago/SNI;\
+    tar -xJf sni-v0.0.95-linux-amd64.tar.xz -C /Archipelago/SNI;\
+    chmod -R 777 /Archipelago/SNI;\
+    rm sni-v0.0.95-linux-amd64.tar.xz
```

Comment:
Does WebHost use SNI?
I would have guessed that WebHost doesn't use SNI.

Reply:
Unsure, I was mainly following docs/running from source.md to get it running. If it is unneeded, that can be removed.